src/olympia/editors/helpers.py

```diff
@@ -449,15 +449,14 @@ def get_actions(self, request):
                 'minimal': True,
                 'comments': False,
             }
-            # Not implemented yet, will be in #5275.
-            # actions['reject_auto_approved'] = {
-            #     'method': self.handler.reject_auto_approved,
-            #     'label': _('Reject'),
-            #     'details': _('This will reject the specified '
-            #                  'auto-approved versions. The comments will '
-            #                  'be sent to the developer.'),
-            #     'minimal': True,
-            # }
+            actions['reject_multiple_versions'] = {
```

Comment:
Hmm... the if on line 438 will only show the `reject_multiple_versions` action if the current_version is auto_approved, right?  Sp this action can't be used after one manual review on an un-auto-approved version has taken place?